xarray/backends/pseudonetcdf_.py

```diff
@@ -1,3 +1,5 @@
+import os
```

Comment:
Please do not import all `os`
```suggestion
import os.path
```